01-Plot-Raster.Rmd

```diff
@@ -107,21 +104,21 @@ plot(DSM_HARV,
 We can view our data "symbolized" or colored according to ranges of values
 rather than using a continuous color ramp. This is comparable to a "classified"
 map. However, to assign breaks, it is useful to first explore the distribution
-of the data using a histogram. The `breaks=` element in the `hist` function
+of the data using a histogram. The `breaks` argument in the `hist()` function
```

Comment:
:+1: got it. thank you for correcting this throughout. I definitely confuse these terms - especially moving between languages.